src/store/channels/index.ts

```diff
@@ -35,6 +35,13 @@ export enum ConversationStatus {
   ERROR,
 }
 
+export enum RoomLabels {
+  WORK = 'm.work',
+  FAMILY = 'm.family',
+  SOCIAL = 'm.social',
```

Comment:
Why are we forcing users to adhere to a restricted set of labels?

Reply:
Yeah if we want to allow the user to create custom labels we shouldn't force them to adhere to a restricted set. Was only planning for requirements now rather than future. Removed restriction & renamed enum to reflect defaults: https://github.com/zer0-os/zOS/pull/2125